app/services/weets.js

```diff
@@ -1,6 +1,14 @@
 const { default: Axios } = require('axios');
+const logger = require('../logger');
+
+const config = require('../../config');
 
 exports.getQuote = async () => {
-  const { data } = await Axios.get('https://geek-jokes.sameerkumar.website/api?format=json');
-  return data;
+  try {
+    const { data } = await Axios.get(config.common.api.randomWeetsApi);
```

Comment:
So that you don't have to do so much nesting you could do something like `const { urlApi } = require ('../../ config'). common.geekJokeApi.`